analysis_options.yaml

```diff
@@ -0,0 +1,2 @@
+analyzer:
```

Comment:
I think we generally prefer using `strong` in the analysis options file - all clients that get it, instead of people knowing to pass in as a cli flag.

But they do end of being the same thing.